prog/weaver/checkpoint.go

```diff
@@ -48,10 +55,11 @@ func checkForUpdates(dockerVersion string, router *weave.NetworkRouter) {
 	}
 	kernelVersion := string(releaseBytes[:i])
 
-	flags := map[string]string{
-		"docker-version": dockerVersion,
-		"kernel-version": kernelVersion,
-	}
+	flags := make(map[string]string)
```

Comment:
I don't understand the need for this change.

Reply:
Reverted this change. Not required.